### DISCUSSION TITLE
Project Status/Future?

### DISCUSSION THREAD

**Discussion by cyberbeast** [go-webauthn/webauthn#162] [Q&A] [answered]
Couldn't find much info about what the future of this project, or the current status? I am scouring the OSS landscape for WebAuthn in Go, and this seems the most promising.

**Comment by james-d-elliott** [comment, marked as answer]
We are relatively active and in a semi-stable state. It's likely that there will be a breaking change before 1.0.0 and there have been a couple thus far however they were necessary to match the spec correctly. This library is used by quite a few large projects and it was originally designed by someone who is now on the W3C WG for WebAuthn.

We have actually also been involved in identifying an issue with the MDS3 blob and are active in the relevant W3C communities.

As far as the future I would really love to be able to improve on the extensibility of this library in particular having direct support for the available extensions in a more digestible way for implementers. The trust anchor validation is also relatively high on my priorities list.

**Reply by cyberbeast** [reply]
Thank you! As a hobbyist I am glad to see this response. I'll play around more with this in my spare time, but I am hoping to get a working example running on my end before diving in deeper.
